src/main/java/se/repos/indexing/standalone/IndexingDaemon.java

```diff
@@ -46,7 +46,7 @@
 	protected Injector global;
 	
 	protected Map<File, CmsRepository> known = new HashMap<File, CmsRepository>();
-	protected Map<CmsRepository, ReposIndexing> loaded = new LinkedHashMap<CmsRepository, ReposIndexing>();
+	protected Map<CmsRepository, ReposIndexing> loaded = new TreeMap<CmsRepository, ReposIndexing>(new CmsRepositoryComparator());
```

Comment:
Maybe change this declaration from `Map` to `SortedMap`, to clarify intent?

Reply:
Done